src/wp-includes/general-template.php

```diff
@@ -4906,6 +4906,38 @@ function __checked_selected_helper( $helper, $current, $echo, $type ) { // phpcs
 	return $result;
 }
 
+/**
+ * Assigns a visual indicator for required form fields.
+ *
+ * @since 6.1.0
+ *
+ * @return string Indicator glyph wrapped in a `span` tag.
+ */
+function wp_required_field_indicator() {
+	/* translators: Character to identify required form fields. */
```

Comment:
maybe we could have something like this?
```
/* translators: Character to identify required form fields. */
$indicator = '<span class="required" aria-hidden="true">' . esc_html__( '*' ) . '</span>';
```
Not sure about that myself, but worth asking :)

Reply:
I see, thanks :)